collections.md

```diff
@@ -344,10 +352,11 @@ You may even pass a callback as the default value. The result of the callback wi
 <a name="method-groupby"></a>
 #### `groupBy()` {#collection-method}
```

Comment:
That's how I did it originally. Taylor changed it. I guess he likes this better.